scheduler/test/cook/test/mesos/dru.clj

```diff
@@ -89,8 +89,64 @@
             ordered-drus [1.0 1.0 1.0 1.5 4.0 5.5]]
         (is (= ordered-drus
                (map (comp :dru second)
-                    (dru/sorted-task-scored-task-pairs (map-vals (partial sort-by identity (util/same-user-task-comparator))
-                                                          (group-by util/task-ent->user task-ents))
-                                                {"ljin" share "wzhao" share "sunil" share}))))))))
+                    (dru/sorted-task-scored-task-pairs
+                      {"ljin" share "wzhao" share "sunil" share}
```

Comment:
These are values from a pre-existing test. I didn't want to change the names used in this test as I just reordered the arguments. We can do that as a separate pass for all tests.